xee/ext_integration_test.py

```diff
@@ -22,6 +22,10 @@
 import numpy as np
 import xarray as xr
 from xarray.core import indexing
+import os
```

Comment:
A couple import suggestions:

- `os` has already been imported. Please remove.
- `tempfile` is a built-in Python module. Please move it up below `pathlib`.
- `rioxarray` is unused. Please remove.
- `rasterio` is not defined as a test dependency. Please include it in `project.optional-dependencies` `tests`.

Reply:
Will do for `os` `tempfile` and `rasterio`

However, from the[ rioxarray docs](https://corteva.github.io/rioxarray/html/getting_started/getting_started.html): 

> "rioxarray extends xarray with the rio accessor. The rio accessor is activated by importing rioxarray like so: `import rioxarray`"

so without that import, `ds.rio.to_raster(...)` etc. fail with:

`AttributeError: 'Dataset' object has no attribute 'rio'`

I can add rioxarray as a test dependency along with rasterio though.